airflow/dags/ingestion/ask-astro-load-airflow-docs.py

```diff
@@ -37,7 +48,7 @@ def ask_astro_load_airflow_docs():
 
     extracted_airflow_docs = task(airflow_docs.extract_airflow_docs)(docs_base_url=airflow_docs_base_url)
 
-    split_md_docs = task(split.split_html).expand(dfs=[extracted_airflow_docs])
+    # split_docs = task(split.split_list)(urls=extracted_airflow_docs, chunk_size=2)
```

Comment:
Should we remove the code here? Or is this comment needed for some reason?

Reply:
No, reason. removed